Control/lib/api.js

```diff
@@ -19,6 +19,8 @@ const config = require('./config/configProvider.js');
 
 // middleware
 const {minimumRoleMiddleware} = require('./middleware/minimumRole.middleware.js');
+const {addDetectorIdMiddleware} = require('./middleware/addDetectorId.middleware.js');
+const {DetectorIdState} = require('./common/lock/detectorIdState.enum.js');
```

Comment:
Hi @pepijndik , what is the reason for the `DetectorId` to include `State`?

For example in the test files you refer to it `DetectorId` https://github.com/AliceO2Group/WebUi/pull/2672/files#diff-dd46489f0cf6038851352a0dcca80206a492f27d97623897f7b65f867f4fb35eR19

Reply:
Fixed  this in New Commit